### ISSUE TITLE
Mouse use not working on second screen

### ISSUE THREAD

**Issue by psarumo** [rcaelers/workrave#276]
The time of use and movement of the mouse is not incremented when the cursor is on the second screen (although the number of clicks increase when I click on each one of my screens and the break window also appears on both monitors).

I am using windows in my laptop +  a monitor.

**Comment by rcaelers** [comment]
I assume you are referring to the 'Mouse usage' and 'Move movement' in the statistics?

**Comment by psarumo** [comment]
Yes, I am. I'm not using the english version, so I didn't know the correct translation
